src/grt/BUILD

```diff
@@ -0,0 +1,154 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2025, The OpenROAD Authors
+
+load("//bazel:tcl_encode_or.bzl", "tcl_encode")
+load("//bazel:tcl_wrap_cc.bzl", "tcl_wrap_cc")
+
+package(
+    default_visibility = ["//:__subpackages__"],
```

Comment:
How is this different than `//visibility:public` ?  It seems to apply to everything.

Reply:
With public, everyone using openroad as a dependency can see also this target, while this makes sure it is limited to openroad deps.